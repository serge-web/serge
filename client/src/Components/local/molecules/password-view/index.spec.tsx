/* global it expect */

import React from 'react'
import renderer from 'react-test-renderer'

import PasswordView from './index'
const onChange = (nextValue: string): void => { console.log(nextValue) }

it('password-view renders correctly', () => {
  const tree = renderer
    .create(<PasswordView onChange={onChange} />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
