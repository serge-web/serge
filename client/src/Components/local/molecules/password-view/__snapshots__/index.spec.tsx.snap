// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`password-view renders correctly 1`] = `
<div
  className="main"
  title="Click to Edit"
>
  <section
    className="password-mask"
  >
    <div
      className="dots"
    />
  </section>
  <div
    className="key"
    onClick={[Function]}
  >
    <svg
      className="svg-inline--fa fa-key"
    />
  </div>
</div>
`;
